api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/argumentAxis/categories.md

```diff
@@ -144,4 +199,6 @@ To sort the arguments, for example, alphabetically, you need to assign an array
 
 ---
 
-[note]Arguments missing from the **categories** array will be added to its end automatically.
+Arguments of the `number` and `date` types on discrete axes are sorted in ascending order regardless of the order they have in the data source. Do not specify the **categories** option for these argument types.
```

Comment:
Is is forbidden to specify `categories` in this case, or are categories simply ignored?